<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LastTransactionsWindow</class>
 <widget class="QWidget" name="LastTransactionsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>852</width>
    <height>533</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>852</width>
    <height>533</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>852</width>
    <height>533</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Historia Transakcji</string>
  </property>
  <widget class="QTableWidget" name="TransactionsTable">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>851</width>
     <height>351</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>851</width>
     <height>351</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <attribute name="horizontalHeaderCascadingSectionResizes">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>141</number>
   </attribute>
   <column>
    <property name="text">
     <string>Imie</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nazwisko</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Data</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rodzaj Transakcji</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Stan Konta Przed Transakcją</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>8</pointsize>
     </font>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Stan Konta Po Transakcji</string>
    </property>
    <property name="font">
     <font>
      <family>Calibri</family>
      <pointsize>10</pointsize>
     </font>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="BackButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>460</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Powrót</string>
   </property>
  </widget>
  <widget class="QLabel" name="TransactionsLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>5</y>
     <width>871</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Historia Transakcji</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="RefreshButton">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>460</y>
     <width>170</width>
     <height>60</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Odśwież</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
